cmd/server.go

```diff
@@ -79,6 +79,21 @@ var serverCmd = &cobra.Command{
 		if err != nil {
 			log.Fatalf("error writing hosts file: %v", err)
 		}
+
+		rootCert, err := ioutil.ReadFile(serverCert)
+		if err != nil {
+			log.Fatalf("error reading %s file: %v", serverCert, err)
+		}
+		err = ioutil.WriteFile(path.Join(configDir, "root-certificate.pem"), rootCert, 0644)
+		if err != nil {
+			log.Fatalf("error writing root-certificate.pem file: %v", err)
+		}
+
+		// FIXME: this is going away as part of fixing https://github.com/lf-edge/adam/issues/16
+		err = ioutil.WriteFile(path.Join(configDir, "Force-API-V1"), []byte{}, 0644)
```

Comment:
Should these not go into `eve-embedded.sh` rather than here? This `/config/Force-API-V1` file is only in the embedded context, and only makes sense to eve itself. 

I don't object to adding some capabilities to adam itself, so that we can eventually get rid of `eve-embedded.sh` (I was thinking about it, actually), but let's not put them in in an unstructured way. This whole thing can be done in a single `cp` command and a single `echo` command in `eve-embedded.sh`

Reply:
No it actually has to go everywhere -- not just into embedded. Otherwise there's absolutely no way for us to signal to the user that this needs to be somehow handed over to EVE via its config.

In fact, that the whole point of Adam generating an EVE config -- it needs to have a way to signal of what options need to be there for it to actually function.

Hence I think it actually belongs here -- the only other alternative @deitch is to completely drop this EVE configuration generation -- 'cuz it really has to be complete or it needs to be dropped.

The thing we don't want it to be is misleading -- that's the worst option of all.

Reply:
Adam already was writing out an eve-compatible config to a local directory when run in `adam server` mode. Right. In that case, this makes sense.